cmd/img_mgmt/port/zephyr/src/zephyr_img_mgmt.c

```diff
@@ -232,11 +246,15 @@ img_mgmt_impl_write_pending(int slot, bool permanent)
 {
     int rc;
 
-    if (slot != 1) {
+    if ((slot != 1) && (slot != 3)) {
```

Comment:
Can't we use `IMG_MGMT_UPDATABLE_IMAGE_NUMBER` here also ?

Reply:
> 
> 
> Can't we use `IMG_MGMT_UPDATABLE_IMAGE_NUMBER` here also ?

The PR has been moved here: https://github.com/apache/mynewt-mcumgr/pull/135